apps/ai-pipeline/.env.example

```
@@ -33,18 +23,19 @@ IS_PRODUCTION=false
 ADMIN_PASSWORD=change_me
 
 # MongoDB (game-engine 과 공유; news-article / scenario-tree contract)
-MONGODB_URL=mongodb://localhost:27017
-MONGODB_DB=safe_or_scam
+MONGODB_URL=mongodb://sos_user:localpass123@localhost:27017/?authSource=admin
+MONGODB_DB=sos_db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGODB_DB key should go before the MONGODB_URL key

(UnorderedKey)